tools/dissector/sample_manager.h

```diff
@@ -34,6 +34,7 @@ extern "C" {
 #include <string>
 #include <map>
 #include <vector>
+#include <forward_list>
```

Comment:
This is a C++11 feature [1].

This break the build here:
In file included from /usr/include/c++/5/forward_list:35:0,
                 from ../../tools/dissector/sample_manager.h:37,
                 from packet-opendds.cpp:10:
/usr/include/c++/5/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support must be enabled with the -std=c++11 or -std=gnu++11 compiler options.


[1] http://www.cplusplus.com/reference/forward_list/forward_list

Reply:
Thank you for catching that, I would have assumed the build system was defaulting to C++03 on my end and I missed that was 11 in the documentation.